ci/container/container-run.sh

```diff
@@ -126,7 +126,9 @@ if [ "$(id -u)" = "1000" ]; then
     if [ -e "${HOME}/.bash_history" ]; then
         PODMAN_RUN_ARGS+=(
             --mount type=bind,source="${HOME}/.bash_history",target="/home/ubuntu/.bash_history"
-        )
+        ) # privileged rootful podman is required due to requirements of IC-OS guest build
+        # additionally, we need to use hosts's cgroups and network
+
```

Comment:
This comment doesn't seem to have anything related to `.bash_history`. Why write it here?

Reply:
Merge error.  Fixed.

Reply:
Also resolved.